engine/src/flutter/shell/platform/windows/platform_handler.cc

```diff
@@ -517,4 +516,22 @@ void PlatformHandler::HandleMethodCall(
   }
 }
 
+WNDCLASS PlatformHandler::RegisterWindowClass() {
+  window_class_name_ = L"FlutterPlatformHandler";
```

Comment:
Should/can we hoist this as a global value in this file? I don't feel strongly, feel free to skip.